internal/commands/util/pr.go

```diff
@@ -226,6 +272,13 @@ func updateAPIURLForGitlabOnPrem(apiURL string) string {
 	return gitlabCloudURL
 }
 
+func updateAPIURLForAzureOnPrem(apiURL string) string {
```

Comment:
I think you should change this function name to: getAPIUrl.
you don't use it just for onPrem and it's not necessarily being updated